--- origsrc/cgilua-5.1.1/examples/config.lua	2007-10-31 15:50:53.000000000 -0500
+++ src/cgilua-5.1.1/examples/config.lua	2008-02-13 15:23:57.546875000 -0600
@@ -4,7 +4,7 @@
 -- You may want to use parts of the this file in your CGILua configuration
 --
 --
-
+require ('cgilua')
 -- Emulating previous CGILua behavior loading file "env.lua" from the script's directory
 --[[
 cgilua.addopenfunction (function ()
@@ -37,7 +37,7 @@
 -- and contain the absolute path of the application's Lua-library directory.
 --[[
 local app_lib_dir = {
-	["/t/"] = "/usr/local/src/cgilua/tests",
+	["/cgi/"] = "/var/www/cgi",
 }
 local package = package
 cgilua.addopenfunction (function ()
@@ -46,4 +46,4 @@
 		package.path = app.."/?.lua;"..package.path
 	end
 end)
---]]
\ No newline at end of file
+--]]
